tools/docker/alertmanager/alertmanager.yml

```diff
@@ -0,0 +1,15 @@
+
+route:
+  receiver: 'mail'
+  repeat_interval: 4h
+  group_by: [ alertname ]
+
+
+receivers:
```

Comment:
how will these alerts work?

Reply:
This is only useful in case you have prometheus alerts created by you. I actually don't use alertmanager and observability-lib is creating grafana managed alert rules. Could be removed or optional depending on a flag